npm-scripts.js

```diff
@@ -1,177 +1,209 @@
-/* eslint-disable no-console */
-
 const process = require('process');
 const os = require('os');
 const fs = require('fs');
 const { execSync, spawnSync } = require('child_process');
-const { version } = require('./package.json');
+const fetch = require('node-fetch');
+const tar = require('tar');
+
+const PKG = JSON.parse(fs.readFileSync('./package.json').toString());
```

Comment:
This is part of my discarded attempt to move to ESM. Still I think it's not a bad idea. For instance, if you `require('package.json')` the whole JSON will be present in the bundled JS (in case of apps or libs that expose a bundle).